build.gradle

```diff
@@ -23,7 +23,8 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.3.3'
+        classpath 'com.android.tools.build:gradle:3.0.1'
```

Comment:
Is it necessary to upgrade the android plugin to 3.0.1 for this plugin?

Reply:
Not necessary will revert back to android plugin to 2.3.3